tests/functional.mk

```diff
@@ -84,14 +84,10 @@ include makelib/build_go_bin.mk
 # 2 - aci directory
 # 3 - ace validator
 define FTST_GENERATE_ACE_IMAGE
+
 $$(call forward-vars,$1,ACTOOL)
 $1: $2/manifest $2/rootfs/ace-validator | $2/rootfs/opt/acvalidator
-	# Set a consistent timestamp so we get a consistent hash
-	# TODO(jonboulle): make this cleaner..
```

Comment:
I don't think we need it. The functional test passed for me without it.

/cc @jonboulle